docs/OV_Runtime_UG/deployment/conditional_compilation_deployment.md

```diff
@@ -0,0 +1,289 @@
+# OpenVINO conditional compilation for optimal binary size{#opevino_conditional_compilation_deploying_guide}
+
+@sphinxdirective
+
+Conditional compilation can significantly reduce the binary size of OpenVINO™ package by excluding unnecessary components for inference of particular models, which makes the application benefit the optimal binary size.
```

Comment:
We can delete "which makes the application benefit the optimal binary size", and state this instead -- This is particularly useful for edge and client deployment scenarios, where reducing the size of application binary is important.